helpers/output_helper.rb

```diff
@@ -0,0 +1,113 @@
+module OutputHelper
+  def greeting_message
+    puts I18n.t(:greeting_message)
+  end
+
+  def greeting_main_menu_message(name)
+    # puts I18n.t(:welcome, name: name)
+    puts I18n.t(:greeting_main_menu_message, name: name)
+  end
+
+  def create_card_message
+    puts I18n.t(:create_card_message)
+  end
+
+  def cards_list_message(cards)
+    cards.each_with_index { |card, i| puts "- #{card.number}, #{card}, #{I18n.t(:press)} #{i + 1}" }
+    puts I18n.t(:press_exit)
+  end
+
+  def delete_card_message(cards)
+    puts I18n.t(:proposal_to_delete)
+    cards_list_message(cards)
+  end
+
+  def no_account_message
+    puts I18n.t(:no_account_message)
+  end
+
+  def no_cards_message
+    puts I18n.t(:no_cards_message)
+  end
+
+  def wrong_number_message
+    puts I18n.t(:wrong_number_message)
```

Comment:
A lot of method from this file looks like
```ruby
def wrong_number_message
    puts I18n.t(:wrong_number_message)
end
```
You can just write in code `puts I18n.t(:wrong_number_message)`
For me, it little bit strange call method which contains one easy line

You can just miss it message)

Reply:
I thought about it and decided to put all the messages in one place. Maybe you're right, let's wait for the mentor's verdict